maintainers.md

```diff
@@ -0,0 +1,64 @@
+# Maintainers
+
+## Role and Responsibilities
+
+The main duties of lesson maintenance include 
+
+* reviewing and merging new pull requests
+* opening and tracking issues
+* guiding the direction of the lesson
+* keeping the lesson up to date with the standard template
+* participating in discussions of lesson content/formatting on the [maintainers mailing list][maintainers]
+* maintain a welcoming and positive atmosphere for contribution by 
+	* responding promptly to new pull requests or issues
+	* following Software Carpentry's [Code of Conduct][CoC]
+
+The lesson maintainer is not required 
+to submit their own changes to the lesson in response to particular issues, 
+but they are welcome to submit PRs as they choose, like any other community 
+member.  
+
+## New Maintainer Selection Process
+
+* Outgoing maintainer emails [discuss][discuss] with announcement of opportunity: 
```

Comment:
When the reference label is the same as the text, you can drop the reference label (look for “implicit link name” in [the spec][1]).  So this can just be `[discuss][]`.

[1]: https://daringfireball.net/projects/markdown/syntax#link